src/sentry/lang/native/symbolicator.py

```diff
@@ -142,7 +142,7 @@ def _process(self, create_task):
                 # retrying after just a second.
                 #
                 # If there is no response attached, it's a connection error.
-                raise RetrySymbolication(retry_after=10)
+                raise RetrySymbolication(retry_after=3)
```

Comment:
10 seconds seems a bit too much, I'd rather see us retry more often even in rare 502/503 cases.

Reply:
Can this just be `SYMBOLICATOR_MAX_RETRY_AFTER`?

Reply:
Yeah, I introduced the `SYMBOLICATOR_MAX_RETRY_AFTER` only after this change, so this is a good idea.